helmfile/values/falco/falco-common.yaml.gotmpl

```diff
@@ -12,6 +12,12 @@ falco:
   grpc_output:
     enabled: true
 
+  rules_file:
+    - /etc/falco/falco_rules.yaml
+    - /etc/falco/falco-incubating_rules.yaml
+    - /etc/falco/falco-sandbox_rules.yaml
+    - /etc/falco/falco_rules.local.yaml
+    - /etc/falco/rules.d
```

Comment:
Is it fine to specify all of these even when you have disabled some of them?

Reply:
It is fine in the sense that nothing breaks. It will create empty files in the containers, so maybe we want some checks here as well

Reply:
https://github.com/elastisys/compliantkubernetes-apps/pull/1843/commits/83a19273b9f21e2076c7981e35a5d9a0b240080f#diff-e91a3a06c409a58ed8e2eb4e6dbe9e8931306dedb3d4c09756344e8636dacf8bR15-R26

Reply:
If it doesn't break anything then it's fine, but it does look better with the checks :+1: